src/assets/scss/_footer.scss

```diff
@@ -0,0 +1,216 @@
+footer {
+  background: url(../images/christmas_snow_trees-wide.png) no-repeat;
+  position: relative;
+  background-position: bottom center;
+  z-index: 0;
+}
+.contacts {
+  &::before {
+    background: url(../images/footer.png) no-repeat;
+    content: "";
+    position: absolute;
+    background-size: cover;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+
+  &__info {
+    padding: 112px 0 122px;
+    display: flex;
+    justify-content: space-between;
+    color: #c7cccf;
```

Comment:
Please use variables

Reply:
Ok, I fixed it. But almost all colors is unique
``` suggestion
$black: #181919;
$white: #f8fafa;
$white2: #ffffff9a;
$gray: #929393;
$blue: #6df6eb;
$bluelight: #b2f2ee;
$bluedark: #97cecb;
```